core/sdk-core/src/main/java/software/amazon/awssdk/core/SdkNumber.java

```diff
@@ -27,20 +27,23 @@
  * An in-memory representation of Number being given to a service or being returned by a service.
  * This is a SDK representation of a Number. This allows conversion to any desired numeric type by providing constructors
  * as below
- * @see #fromBigDecimal(BigDecimal) to create from a BigDecimal.
- * @see #fromBigInteger(BigInteger) to create from a BigInteger.
- * @see #fromDouble(double) to create from a double
- * @see #fromFloat(float) to create from a float.
- * @see #fromLong(long) to create from a long.
- * @see #fromShort(short) to create from a short.
- * @see #fromInteger(int) to create from an integer.
- * @see #fromString(String)  to create from a Stringl,
- * Thus by doing this, this class is able to preserve arbitrary precison of any given number.
+ * <ul>
```

Comment:
What was the problem with using `@see`?

Reply:
`@see` tags are to generate `See Also` sections generally linking to documents or API outside the current one, not links in general, that's what `{@link …} is designed for usually below the main document, as in 

---
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

See Also [Lorem ipsum wikipedia entry](https://en.wikipedia.org/wiki/Lorem_ipsum)

---

Using `@see` here also produces a warning about `warning: Tag {@link}: nested link. If {@link SdkNumber} is expected in a particular number format then its corresponding getter methods can be used.` given that there's leading content below the **See Also** section of the docs.

Reply:
See also the way it is currently rendered [here](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/core/SdkNumber.html).